### DISCUSSION TITLE
Possibility to choose audio

### DISCUSSION THREAD

**Discussion by MorGotHMGH** [jely2002/youtube-dl-gui#303] [Ideas]
Hi, mate please make possibility to choose audio codec when downloading wideo and audio, now when movie is downloaded audio codec is opus, please make possibility to choose audio codec.

**Comment by arax-1238** [comment]
I woud be a great thing, as my old TV do no support Opus codec, but only aac o mp3...

**Comment by jely2002** [comment]
This is supported since version 3.2.0, you can select the codec per video by clicking on the "tools" icon. Optionally you can always show the codec select by adding it as an "additional selector" in the app's appearance settings.
